std/process.d

```diff
@@ -2212,14 +2233,14 @@ class ProcessException : Exception
 
 
 /**
-Determines the path to the current user's default command interpreter.
+Determines the path to the current user's preferred command interpreter.
 
 On Windows, this function returns the contents of the COMSPEC environment
-variable, if it exists.  Otherwise, it returns the string $(D "cmd.exe").
+variable, if it exists.  Otherwise, it returns the result of $(LREF nativeShell).
 
 On POSIX, $(D userShell) returns the contents of the SHELL environment
-variable, if it exists and is non-empty.  Otherwise, it returns
-$(D "/bin/sh").
+variable, if it exists and is non-empty.  Otherwise, it returns the result of
+$(LREF nativeShell).
```

Comment:
This comment says "it returns the result of `nativeShell`," but the various shells are actually hard-coded in: any reason why?  I'm preparing a PR for Android, any reason not to have the code below actually do what the comment says?  If not, I'll make `userShell` call `nativeShell`, so we don't have the same list of shells duplicated in two methods.